shell/initialize/client.js

```diff
@@ -637,6 +668,7 @@ function fixPrepatch(to, ___) {
   });
 }
 
+// more nuxt stuff ??????
```

Comment:
plumbing probably related to page lifecycle events. Also plugs in to vue router changed events, so guessing is required for middleware